cli/src/describe.rs

```diff
@@ -0,0 +1,34 @@
+use anyhow::{Error, Result};
+use clap::Parser;
+use model::clients::CrdClient;
+use model::test_manager::TestManager;
+use model::CrdExt;
+
+/// Retrieve the YAML description of a test or resource.
+#[derive(Debug, Parser)]
+pub(crate) struct Describe {
+    /// The name of the test we want the description from.
```

Comment:
nit: The cli saying `we` in the help messaging is a bit odd. Should be more of a "statement":

```
The name of the test to return the yaml for
```